osidb/tests/test_endpoints.py

```diff
@@ -1258,6 +1258,52 @@ def test_flaw_update_collision(self, auth_client, test_api_uri):
             "It has been probably edited by someone else in the meantime"
         )
 
+    def test_flaw_comment_create(self, auth_client, test_api_uri):
+        """
+        Test that adding a flaw comment by sending a POST request works.
+        """
+
+        def new_flaw():
+            flaw = FlawFactory(embargoed=False)
+            AffectFactory(flaw=flaw)
+            response = auth_client.get(f"{test_api_uri}/flaws/{flaw.uuid}")
+            assert response.status_code == 200
+            assert not FlawComment._base_manager.filter(flaw=flaw).exists()
+            return flaw
+
+        def get_response(flaw, new_comment):
```

Comment:
A better name would be `new_comment` or `create_comment`. `get_response` is too general.

Reply:
Resolved in commit a3415f9b5cde43c7e4fb3ac8476e8791fd710365